index.html

```diff
@@ -19,7 +20,13 @@ <h5>9:41</h5>
     </div>
     <div class="lower-nav">
       <h3>Tarek Yosry</h3>
-      <img src="./images/Union.png" alt="hamburger menu">
+      <img class="hamburger-menu" src="./images/Union.png" alt="hamburger menu">
+      <ul class="pop-up">
+        <li><a href="">Portfolio</a></li>
+        <li><a href="">About</a></li>
+        <li><a href="">Contact</a></li>
+      </ul>
```

Comment:
- [ ] You did a good job on the mobile menu functionalities but there are some issues you need to fix. Please use correct color `#FFF5E1` for mobile menu options.